ICAFIX/scripts/VolumeSmoothSICA.sh

```diff
@@ -0,0 +1,13 @@
+SubjectFolder="${1}"
+Subject="${2}"
+fMRIName="${3}"
+HighPass="${4}"
+Resolution="${5}"
+SmoothingFWHM="${6}"
+Caret7_command="${7}"
+
+SmoothingSigma=`echo "$SmoothingFWHM / (2 * sqrt(2 * l(2)))" | bc -l`
+
+${Caret7_command} -volume-label-import ${SubjectFolder}/MNINonLinear/ROIs/VolumeSmoothROIs.${Resolution}.nii.gz "" ${SubjectFolder}/MNINonLinear/ROIs/VolumeSmoothROIs.${Resolution}.nii.gz
+${Caret7_command} -volume-parcel-smoothing ${SubjectFolder}/MNINonLinear/Results/${fMRIName}/${fMRIName}_hp${HighPass}.ica/filtered_func_data.ica/melodic_oIC.nii.gz ${SubjectFolder}/MNINonLinear/ROIs/VolumeSmoothROIs.${Resolution}.nii.gz ${SmoothingSigma} ${SubjectFolder}/MNINonLinear/Results/${fMRIName}/${fMRIName}_hp${HighPass}.ica/filtered_func_data.ica/melodic_oIC_s4.nii.gz
```

Comment:
The output is hardcoded as "melodic_oIC_s4"?  That seems silly (the "_s4" probably meant "smooth with a kernel of 4mm"), it should probably be an argument, as should the input data file.

Assuming this script exists because this is done multiple times (looks like only from matlab), that means it does the -volume-label-import multiple times, does the `VolumeSmoothROIs` file change in between them?  This is a very small, specialized script, which suggests it may fit better as a function than a separate file.  `l()` is natural log in bc, the rest should be obvious.

Reply:
I will try not to use a file or function but code them in the compute feature loop.